app/models/file_search.rb

```diff
@@ -35,32 +38,32 @@ def search_term
     params[:q]
   end
 
-  def work_name
-    params[:work]
+  def page
+    params[:page]
   end
 
-  def venue_names
-    params.fetch(:venues) { all_venue_names }
+  def per_page
+    params[:per_page]
   end
 
-  def record_types
-    params.fetch(:record_types) { all_record_types }
+  def work_name
+    params[:work]
   end
 
-  def show_articles?
-    record_types.include?("articles")
+  def venue_names
+    params.fetch(:venues) { all_venue_names }
   end
 
-  def show_images?
-    record_types.include?("images")
+  def resource_types
+    resource_type.present? ? [resource_type.to_s] : []
   end
 
-  def show_audios?
-    record_types.include?("audios")
+  def types
+    params.fetch(:types) { all_resource_types }
   end
 
-  def show_videos?
-    record_types.include?("videos")
+  def show?
+    types.include?(resource_type.to_s.pluralize)
```

Comment:
Is `pluralize` needed here?  I thought we either had lowercase-plural or capitalized singular, depending on the domain.  only pluralizing without downcasing seems incomplete or unnecessary.